examples/src/test/java/cucumber/runtime/arquillian/ClientFeatureLoaderTest.java

```diff
@@ -1,97 +1,97 @@
-package cucumber.runtime.arquillian;

```

Comment:
I had a some issues with trying to run the examples, so my changeson those files could be ignored, let me try to restore them to their original form